activesupport/test/core_ext/hash/transform_keys_test.rb

```diff
@@ -21,12 +21,14 @@ class TransformKeysTest < ActiveSupport::TestCase
   test "transform_keys returns an Enumerator if no block is given" do
     original = { a: 'a', b: 'b' }
     enumerator = original.transform_keys
+    assert_nil enumerator.size
```

Comment:
I think `Enumerator#size` is the number of times it'll *yield*, not the size of whatever might get returned. So, `origin.transform_keys.size #=> 2`.